src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+    @Test
+    public void checkNull() {
```

Comment:
test cases should also specify the name of the method under test in the method name. For example, `isAnyNull_emptyList_falseReturned()`.